### ISSUE TITLE
Changing default zoom to below 100% changes UI Scale to smaller but doesn't show in the setting

### ISSUE THREAD

**Issue by srirambv** [brave/browser-laptop#9334]
- Did you search for similar issues before submitting this one?
Yes

- Describe the issue you encountered:
Changing default zoom to below 100% changes UI Scale to smaller but doesn't show in the setting

- Platform (Win7, 8, 10? macOS? Linux distro?):
Found on Windows 10 x64. Needs verification on macOS (cc: @alexwykoff @LaurenWags @luixxiul )

- Brave Version (revision SHA):
Brave	0.16.4
rev	15d8d36
Muon	4.0.0

- Steps to reproduce:
    1. Clean install 0.16.4
    2. Enable Always show URL bar from general settings
    3. Set Default zoom to 90%, visually the UI gets smaller including the URLbar and buttons
    4. Change UI Scale to smaller, No visible changes to URL bar in terms of size
    5. Change UI Scale back to Normal, URL bar size is changed to default
    6. Change UI Scale back to smaller , visible URL bar size change
    
- Actual result:
Changing default zoom to below 100% changes UI Scale to smaller but doesn't show in the setting

- Expected result:
If default zoom is set to below 100% and UI scale is auto changed then the setting has to indicate it

- Will the steps above reproduce in a fresh profile? If not what other info can be added?
Yes

- Is this an issue in the currently released version?
Yes

- Can this issue be consistently reproduced?
Yes

- Extra QA steps:
    1.
    2.
    3.

- Screenshot if needed:
![ui](https://user-images.githubusercontent.com/17010094/26920318-921e3046-4c30-11e7-97dd-79b22f4428ff.gif)

- Any related issues:
cc: @bsclifton

**Comment by luixxiul** [comment]
Absolutely that's the regression. Changing the value to `500%` makes the browser completely unusable until clearing the profile.

https://github.com/brave/browser-laptop/issues/8144

The bug can be reproduced on 0.15.312 (d4f169a).

**Comment by luixxiul** [comment]
I put the issue to 0.17.x for now as it can be reproduced on the released version.

**Comment by srirambv** [comment]
The problem is the default zoom is changing the UI zoom level and not just the page zoom.  So first you have to set the UI zoom level and then accordingly change the default page zoom level to adjust to need. 

cc: @bsclifton

**Comment by luixxiul** [comment]
On Debian:

![clipboard01](https://user-images.githubusercontent.com/3362943/27575821-ef961b4c-5b55-11e7-9c09-f3fc99da7f5b.jpg)

I believe this is a regression. Is it not? @bsclifton

**Comment by bsclifton** [comment]
@luixxiul what is the issue exactly? (sorry, I'm not sure if the problem is clearly stated?)

In your screenshot, if you didn't have the dev tools open, you'd be able to see the setting if you had the dev-console closed

**Comment by luixxiul** [comment]
I cannot remember why I pinged you but this is the issue anyway:

![bug](https://user-images.githubusercontent.com/3362943/28391205-fc863476-6d17-11e7-9ff6-7905d1756973.gif)

**Comment by Liunkae** [comment]
Confirmed with Brave 0.17.19 (4e46480) on macOS 10.12.x.

**Comment by bsclifton** [comment]
I don't think this is a bug, per-se... but we should have an easy way to go back.  

Maybe if someone pics an extreme setting like this, we should make the change and then open a confirm dialog... "Did you mean to make this change?".  We could even have the setting revert itself within 10 seconds if the person doesn't respond.

cc: @bradleyrichter for comment

**Comment by luixxiul** [comment]
@bsclifton I have thought that UI scale can be changed via the setting `toolbar and UI elements scale` on `about:preferences#advanced` only like this.

![bug](https://user-images.githubusercontent.com/3362943/28489710-f72b6526-6f03-11e7-9b3a-9e3507bfe995.gif)

While the issue on above https://github.com/brave/browser-laptop/issues/9334#issuecomment-316529245 is that UI scale is changed via `default zoom level`, which is the same issue as reported here: https://github.com/brave/browser-laptop/issues/8144

**Comment by Liunkae** [comment]
This is a bug because the **zoom level should not be affecting the browser UI**.

**Comment by luixxiul** [comment]
The issue above https://github.com/brave/browser-laptop/issues/9334#issuecomment-316529245 is no longer reproduced on 0.18.12

**Comment by Liunkae** [comment]
Still reproducible with Brave 0.20.30 on macOS 10.13.x.
